releases/3.3.0/artifacts.json

```diff
@@ -0,0 +1,212 @@
+{
+  "3.3.0": {
+    "holoscan": {
+      "debian-version": "3.1.0.3-1",
```

Comment:
Note that 3.3.0 will follow the plan:
- Provide libraries built against glibc 2.35 (Ubuntu 22.04) in debian packages, wheels
- Provide release container based on TensorRT 25.03 (Ubuntu 24.04, glibc 2.39)

Depending on Holoscan CLI requirements:
- If Ubuntu 24.04 environment is acceptable, continue to use the Holoscan SDK 3.3.0 dev container
- If Ubuntu 22.04 environment is required, consider using the CUDA or TensorRT 24.08 base container and install the Holoscan apt package